server/plan_replayer.go

```diff
@@ -53,10 +53,25 @@ func (s *Server) newPlanReplayerHandler() *PlanReplayerHandler {
 func (prh PlanReplayerHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	params := mux.Vars(req)
 	name := params[pFileName]
-	path := filepath.Join(domain.GetPlanReplayerDirName(), name)
+	handler := downloadFileHandler{
+		filePath:           filepath.Join(domain.GetPlanReplayerDirName(), name),
+		fileName:           name,
+		infoGetter:         prh.infoGetter,
+		address:            prh.address,
+		statusPort:         prh.statusPort,
+		urlPath:            fmt.Sprintf("plan_replyaer/dump/%s", name),
+		downloadedFilename: "plan_replayer",
+	}
+	handleDownloadFile(handler, w, req)
+}
+
+func handleDownloadFile(handler downloadFileHandler, w http.ResponseWriter, req *http.Request) {
```

Comment:
What about making this function a method of struct `downloadFileHandler`?

Reply:
not necessary I think.